packages/manager/modules/pci/src/projects/project/users/index.js

```diff
@@ -1,22 +1,27 @@
 import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
+import service from '../storages/object-storage/object-storage.service';
```

Comment:
Same as [this comment](https://github.com/ovh/manager/pull/7354/files#r897574863)

Reply:
Taken care